hack/scale_out_poc/config_haproxy/cfg_generator/BUILD

```diff
@@ -0,0 +1,35 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
An updated version of this script has already been moved to master branch in #929. This file should be removed.

Reply:
Is there a corresponding change in scale_out_poc branch to reference the new location?

Reply:
deleted